tests/hooks_test.py

```diff
@@ -57,3 +57,23 @@ def test_python_check_blanket_noqa_positive(s):
 )
 def test_python_check_blanket_noqa_negative(s):
     assert not HOOKS['python-check-blanket-noqa'].search(s)
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        'eval()',
+    ),
+)
+def test_python_check_positive(s):
+    assert HOOKS['python-noeval'].search(s)
+
+
+@pytest.mark.parametrize(
+    's',
+    (
+        'literal_eval()',
+    ),
+)
+def test_python_check_negative(s):
+    assert not HOOKS['python-noeval'].search(s)
```

Comment:
neither of these testcases are valid _calls_ of the two functions -- also if there's only one example there's no need for `parametrize`

Reply:
the first half of this is still an issue `eval()` and `literal_eval()` aren't representative testcases of what you'd see in actual code